src/components/Chart.jsx

```diff
@@ -55,19 +55,15 @@ class Chart extends Component {
                         </RenderInsideChart>
                         <RenderInsideChart>
                             <div className="cq-top-ui-widgets">
-                                <ChartTitle />
                                 <AssetInformation />
                                 <ComparisonList />
                             </div>
                         </RenderInsideChart>
+                        <ChartTitle />
```

Comment:
Also you shouldn't move `<ChartTitle />` out of `<RenderInsideChart>` because indicator panels can be placed on top of the chart